npm/design-system/src/components/SpecList/SpecList.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import {
+  FileComponentProps,
+  FolderComponentProps,
+  FileExplorer,
+  FileExplorerProps,
+} from '../FileExplorer/FileExplorer'
+import { makeFileHierarchy } from '../FileExplorer/helpers/makeFileHierarchy'
+
+import { InlineIcon } from '@iconify/react'
+import javascriptIcon from '@iconify/icons-vscode-icons/file-type-js-official'
+import typescriptIcon from '@iconify/icons-vscode-icons/file-type-typescript-official'
+import reactJs from '@iconify/icons-vscode-icons/file-type-reactjs'
+import reactTs from '@iconify/icons-vscode-icons/file-type-reactts'
+import folderClosed from '@iconify/icons-vscode-icons/default-folder'
+import folderOpen from '@iconify/icons-vscode-icons/default-folder-opened'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { fas } from '@fortawesome/free-solid-svg-icons'
+import styles from './SpecList.module.scss'
+
+library.add(fas)
```

Comment:
Definitely don't want calls like this happening on a per-component basis. Again, fine for now, but this will need to be changed.

Reply:
Yep, we should initialize this once (`index.ts` for example). I will make this change

Reply:
Looks like we aren't using this, I just nuked it

Reply:
This happens one time when module resolved. So it won't be called more and more, but it will be nice to have one entry point that reexport icons

Reply:
> So it won't be called more and more

Correct, but the assumption is that we would repeat this pattern in every component, in which case on resolution, there would be `n` calls.